webrtc.html

```diff
@@ -2350,7 +2331,7 @@ <h3>Interface Definition</h3>
 interface RTCPeerConnection : EventTarget  {
   Promise&lt;RTCSessionDescriptionInit&gt; createOffer(optional RTCOfferOptions options);
   Promise&lt;RTCSessionDescriptionInit&gt; createAnswer(optional RTCAnswerOptions options);
-  Promise&lt;void&gt; setLocalDescription(RTCSessionDescriptionInit description);
+  Promise&lt;void&gt; setLocalDescription(RTCLocalSessionDescriptionInit description);
```

Comment:
Would this be more readable if we said "optional RTCSessionDescription description" instead?

Reply:
@henbos It should say `optional` for sure, thanks for catching that.

We don't require an `RTCSessionDescription` interface however—which would require a browser object as input—because we wanna support people doing tricks like
```js
pc.setLocalDescription({type: "pranswer", sdp: answer.sdp});
```

Reply:
And also
```js
pc.setLocalDescription({type: "rollback"});
```